playbooks/roles/l2tp-ipsec/templates/instructions.md.j2

```diff
@@ -16,6 +16,46 @@ If the connection appears to time out, first check to make sure the passwords we
 
 <a name="windows"></a>
 ### Windows ###
+
+In order to follow the quick directions, you'll need to use Windows PowerShell ISE. To start it:
+
```

Comment:
I think this can be worded better by changing it to something like:

"In order to proceed, you will need to start Windows PowerShell ISE in administrator mode"....